liquibase/liquibase.github.com
# [LOW-2] - Add Favicon to the Liquibase.org site

## What I Did
* Added favicons to liquibase.org site
* added theme information

## How To Test It
* Should see favicon in all browsers
* should see Safari tile when bookmarking
* should see 'theme' on android devices if served on HTTPS
* should be able to 'add to homescreen' on android and ios with name Liquibase and theme.

## Picture of a Cute Animal (Not Required But Recommended for Sanity Purposes)
